src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+* Added new metric `http.server.request.duration` which replaces
+  `http.server.duration` if a user opted into the new semantic convention
+  by setting the `OTEL_SEMCONV_STABILITY_OPT_IN` environment variable.
+  This new metric is emitted as seconds and has unique
```

Comment:
**Note** : The unit of `http.server.request.duration` is `seconds` which has changed from `milliseconds` in `http.server.duration`.  The histogram buckets are adjusted accordingly as per the [spec](link)